CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Development - Contributing
+
+# Requirements
+* [poetry](https://github.com/python-poetry/poetry): For dependency management and packaging in Python
+* [pre-commit][pre-commit]: For identifying code issues before submission to code review
+
+## Developing
+
+To start working on this project, here are some guidelines to set up your environment:
+  1. `git clone https://github.com/anancarv/python-artifactory.git`
+  2. `cd python-artifactory`
+  3. Install poetry: `pip install poetry==0.12.16`
```

Comment:
I wouldn't show this installation method to new users, it's not recommended as it installs Poetry dependencies in the global pip store (which can mess up or be messed up by other apps). Simply redirecting to the [installation docs](https://python-poetry.org/docs/#installation) specifying which version to use would probably be enough (also migrating to Poetry 1.0 would be :fire: ^^)